Added and updated tools and repos

stuhli opened the pull request:
I have added some tools and other repos. I have tried to make sure that these sources have been updated at least within the last six months and have some thematic relevance to the work of incident response folks. I also took the liberty of making some updates and slightly adjusting the structure in some places.

meirwah commented on README.md:
```diff
@@ -104,11 +109,24 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 * [Sandia Cyber Omni Tracker (SCOT)](https://github.com/sandialabs/scot) - Incident Response collaboration and knowledge capture tool focused on flexibility and ease of use. Our goal is to add value to the incident response process without burdening the user.
 * [threat_note](https://github.com/defpoint/threat_note) - Lightweight investigation notebook that allows security researchers the ability to register and retrieve indicators related to their research.
 
+### IOC Management
```
let's remove this section, there are other lists covering this like https://github.com/hslatman/awesome-threat-intelligence

stuhli replied:
done with 7ffb3a998d8aa7663d6b913447b78d15d2a0b03d

meirwah commented on README.md:
```diff
@@ -205,13 +227,18 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 * [PMDump](http://www.ntsecurity.nu/toolbox/pmdump/) - Tool that lets you dump the memory contents of a process to a file without stopping the process.
 
-### Sandboxing/reversing tools
+### Ransomware Mitigation Tools
```
let's not add a section with so little in it

stuhli replied:
done with 7ffb3a998d8aa7663d6b913447b78d15d2a0b03d
moved to https://github.com/meirwah/awesome-incident-response#other-tools

meirwah commented on README.md:
```diff
@@ -220,28 +247,46 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
+* [LOKI](https://github.com/Neo23x0/Loki) - Free IR scanner for scanning endpoint with yara rules and other indicators(IOCs).
+* [Spyre](https://github.com/spyre-project/spyre) - Simple YARA-based IOC scanner written in Go
+
+### Security Orchestration, Automation and Response (SOAR)
```
these should go https://github.com/meirwah/awesome-incident-response#incident-management

stuhli replied:
done with 7ffb3a998d8aa7663d6b913447b78d15d2a0b03d

meirwah commented on README.md:
```diff
@@ -220,28 +247,46 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
+* [DFTimewolf](https://github.com/log2timeline/dftimewolf) - A framework for orchestrating forensic collection, processing and data export.
+* [Shuffle](https://github.com/frikky/Shuffle) - A general purpose security automation platform focused on accessibility.
+
+### Sysmon Config Templates
```
let's not add this one too, not much in it

stuhli replied:
done with 7ffb3a998d8aa7663d6b913447b78d15d2a0b03d
moved to https://github.com/meirwah/awesome-incident-response#other-tools

meirwah commented on README.md:
```diff
@@ -220,28 +247,46 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 * [Yomi](https://yomi.yoroi.company) - Free MultiSandbox managed and hosted by Yoroi.
 
-### Timeline tools
+### Scanner Tools
```
not sure this section makes sense here

stuhli replied:
Unfortunately, I can't think of another section where I could simply move it.

In the incident case, being able to scan systems for IOCs with a stand-alone tool (without EDR) is, in my view, an important task that has to be mastered in IR and which I would therefore look for here in the list. But I am also happy to be convinced of the opposite.

meirwah reviewed (commented):
plz for next time, don't make 1 PR with many changes
split PR to smaller PR's , so it will easier and faster to review

stuhli commented:
Sorry for making you wait so long. Yes, I understand, next time I'll split it.

I will look at the comments and implement. If the merge becomes too confusing due to the long time, I'll close the PR and start over individually. Then also individually.

stuhli commented:
I'm through with my changes for now.

I have implemented your comments - with one exception (see above at the code snippets for details). I have merged the current state from the master and removed the conflict. I also corrected a few small things in the table of contents.

Have a look at it when you get the chance, if it is ok.

meirwah commented:
hey
sorry for getting late on this

Can you update your branch from master ?

stuhli commented:
Hi,
I hope I could solve the conflicts correctly?!

meirwah commented:
will look, for future, plz break your changes to smaller PR's which are much easier to review and merge

meirwah commented on README.md:
```diff
@@ -129,13 +143,15 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 
 * [AppCompatProcessor](https://github.com/mbevilacqua/appcompatprocessor) - AppCompatProcessor has been designed to extract additional value from enterprise-wide AppCompat / AmCache data beyond the classic stacking and grepping techniques.
 * [APT Hunter](https://github.com/ahmedkhlief/APT-Hunter) - APT-Hunter is Threat Hunting tool for windows event logs.
+* [Beagle](https://github.com/yampelo/beagle) - Incident response and digital forensics tool which transforms security logs and data into graphs.
```
this one doesn't seem maintained anymore, remove it plz

stuhli replied:
done with f7d79d80d81a96c63c43dde7555ef47ec235f748
